src/components/Prayer.vue

```diff
@@ -49,8 +57,21 @@ export default {
       locationDetermined: false,
       requestError: false,
       errorMessage: 'Failed to Get Prayer Times',
+
+      cityInput: '',
+      cities: [],
+      previousCity:''
     };
   },
+
+  watch: {
+    // whenever input changes, this function will run
+    cityInput: function (newCity, oldCity) {
+      newCity+oldCity; //to bypass 'no-unused-variables' error
```

Comment:
Since `oldcity` is not used here at all you can just skip defing it as the second parameter.

i.e. `    cityInput: function (newCity) {`


That would fix the no unused variables

Reply:
Thank you for pointing that out. Sorted now